source/_docs/autostart/systemd.markdown

```diff
@@ -77,30 +77,30 @@ WantedBy=multi-user.target
 You need to reload `systemd` to make the daemon aware of the new configuration.
 
 ```bash
-$ sudo systemctl --system daemon-reload
+sudo systemctl --system daemon-reload
 ```
 
 To have Home Assistant start automatically at boot, enable the service.
 
 ```bash
-$ sudo systemctl enable home-assistant@YOUR_USER
+sudo systemctl enable home-assistant@YOUR_USER
 ```
 
 To disable the automatic start, use this command.
 
 ```bash
-$ sudo systemctl disable home-assistant@YOUR_USER
+sudo systemctl disable home-assistant@YOUR_USER
 ```
 
 To start Home Assistant now, use this command.
 ```bash
-$ sudo systemctl start home-assistant@YOUR_USER
+sudo systemctl start home-assistant@YOUR_USER
 ```
 
 You can also substitute the `start` above with `stop` to stop Home Assistant, `restart` to restart Home Assistant, and 'status' to see a brief status report as seen below.
 
 ```bash
-$ sudo systemctl status home-assistant@YOUR_USER
+sudo systemctl status home-assistant@YOUR_USER
```

Comment:
Please restore this one.
If a command + output is shown, the command must be prefixed in order to distinguish the command from the output (as per our documentation guidelines).